onadata/libs/filters.py

```diff
@@ -399,6 +399,19 @@ def filter_queryset(self, request, queryset, view):
         return queryset
 
 
+class ImageAttachmentFilter(filters.BaseFilterBackend):
```

Comment:
Could this class perhaps be renamed to `AttachmentTypeFilter`? It more fits what the filtering is on.

Reply:
Sounds okay. Done.